src/app/shared/components/reject-modal-window/reject-modal-window.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Constants } from 'src/app/shared/constants/constants';
+import { Application } from 'src/app/shared/models/application.model';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  Validators,
+} from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-reject-modal-window',
+  templateUrl: './reject-modal-window.component.html',
+  styleUrls: ['./reject-modal-window.component.scss'],
+})
+export class RejectModalWindowComponent implements OnInit {
+  readonly constants: typeof Constants = Constants;
+
+  modalTitle = 'ВІДМОВИТИ';
+  modalDescription =
+    'Ви впевнені, що хочете перевести заяву у статус ”Відмовлено”?';
+
+  ReasonFormGroup: FormGroup;
```

Comment:
Well, in your case there is one reactive field - description, that's all, so I believe you don't need the formgroup? FormGroup you need usually when you wanna unite several reactive fields in one form, for example, name, surname, date of birth. Then, you can have formgourp where you can track the changing of all the fields. But in your case, there is only one, so probably get rid of FormGroup and use one FormControl.  

```suggestion
  ReasonFormControl= new FormControl('');
```

and then in template, you can remove everything related to the form group and add [formControl]="ReasonFormControl" to the textarea.

Reply:
thanks, it is updated